src/lib/updateGrantsRecipients.js

```diff
@@ -347,6 +445,8 @@ export async function processFiles(hashSumHex) {
           grantSpecialistEmail: valueFromXML(g.grants_specialist_email),
           annualFundingMonth: valueFromXML(g.annual_funding_month),
           inactivationReason: valueFromXML(g.inactivation_reason),
+          geographicRegion: valueFromXML(g.geographic_region),
+          geographicRegionId: parseInt(g.geographic_region_id, 10) || null,
```

Comment:
`??` feels like a better fit here than `||` given you likely don't want a `0` to be treated as falsey.

Reply:
0 isn’t valid right now, but I refactored that line. Used `condition ? parseInt(...) : null` instead of `parseInt() ?? null` since `parseInt `can return `NaN `for empty/missing values, and we want null in the db.

Reply:
Reverting to previous `parseInt(...) || nul`l to avoid inserting `'NaN'` values and preserve correct `null `mapping for invalid ids.